Cubical/Categories/Instances/CommRings.agda

```diff
@@ -35,11 +45,61 @@ _⋆_ CommRingsCategory {R} {S} {T}        = compCommRingHom R S T
 ⋆Assoc CommRingsCategory {R} {S} {T} {U} = compAssocCommRingHom {R = R} {S} {T} {U}
 isSetHom CommRingsCategory               = isSetRingHom _ _
 
+forgetfulFunctor : Functor CommRingsCategory (SET ℓ)
+F-ob forgetfulFunctor R    = R .fst , CommRingStr.is-set (snd R)
+F-hom forgetfulFunctor f x = f .fst x
+F-id forgetfulFunctor      = funExt (λ _ → refl)
+F-seq forgetfulFunctor f g = funExt (λ _ → refl)
+
+open Iso
+
+CommRingIsoIsoCatIso : {R S : CommRing ℓ} → Iso (CommRingIso R S) (CatIso CommRingsCategory R S)
+mor (fun CommRingIsoIsoCatIso e) = (e .fst .fun) , (e .snd)
+inv (fun (CommRingIsoIsoCatIso {R = R} {S}) e) =
+    e .fst .inv
+  , makeIsRingHom (sym (cong (e .fst .inv) (pres1 (e .snd))) ∙ e .fst .leftInv _)
+                  (λ x y → let rem = e .fst .rightInv _
+                                  ∙∙ (λ i → S .snd ._+_ (e .fst .rightInv x (~ i)) (e .fst .rightInv y (~ i)))
+                                  ∙∙ sym (pres+ (e .snd) _ _)
+                            in injCommRingIso {R = R} {S} e _ _ rem)
+                  (λ x y → let rem = e .fst .rightInv _
+                                  ∙∙ (λ i → S .snd ._·_ (e .fst .rightInv x (~ i)) (e .fst .rightInv y (~ i)))
+                                  ∙∙ sym (pres· (e .snd) _ _)
+                           in injCommRingIso {R = R} {S} e _ _ rem)
+sec (fun CommRingIsoIsoCatIso e) = RingHom≡ (funExt (e .fst .rightInv))
+ret (fun CommRingIsoIsoCatIso e) = RingHom≡ (funExt (e .fst .leftInv))
+fun (fst (inv CommRingIsoIsoCatIso e)) = e .mor .fst
+inv (fst (inv CommRingIsoIsoCatIso e)) = e .inv .fst
+rightInv (fst (inv CommRingIsoIsoCatIso e)) x i = fst (e .sec i) x
+leftInv (fst (inv CommRingIsoIsoCatIso e)) x i = fst (e .ret i) x
+snd (inv CommRingIsoIsoCatIso e) = e .mor .snd
+rightInv CommRingIsoIsoCatIso x = CatIso≡ _ _ (RingHom≡ refl) (RingHom≡ refl)
+leftInv (CommRingIsoIsoCatIso {R = R} {S}) x =
+  Σ≡Prop (λ x → isPropIsRingHom (CommRingStr→RingStr (R .snd))
+                                (x .fun)
+                                (CommRingStr→RingStr (S .snd)))
+         (Iso≡Set (is-set (snd R)) (is-set (snd S)) _ _ (λ _ → refl) (λ _ → refl))
+
+isUnivalentCommRingsCategory : ∀ {ℓ} → isUnivalent {ℓ = ℓ-suc ℓ} CommRingsCategory
```

Comment:
@ecavallo This is what I was talking about yesterday. To make this less annoying to prove I introduced `CommRingIso` which hopefully will be useful for other things